### PR TITLE
report: Add virresh's GSoC report

### PR DESCRIPTION
**Pull request by virresh**
https://deploy-preview-617--coala-projects.netlify.com/gsoc/2018/statusreport/virresh

### CODE REVIEW

**Comment by virresh** [comment]
https://deploy-preview-617--coala-projects.netlify.com/gsoc/2018/statusreport/virresh

File: reports/virresh.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+ - Gitter:
+   - username: virresh
+   - link: https://gitter.im/virresh
+ - Openhub:
+   - username: virresh
+   - link: https://www.openhub.net/accounts/virresh
+email: viresh16118@iiitd.ac.in
+blog: https://virresh.wordpress.com/
+activity:
+ - 0:
```
Comment (jayvdb) [inline reports/virresh.md]: See changes requested at https://github.com/coala/projects/issues/615

File: _reports/virresh.md [hunk 2]
```diff
@@ -0,0 +1,391 @@
...
+      - details: >
+          "Initial Commit"
+  - 4:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/168263a2
+      - details: >
+          "ci: Integrate inital ci test"
+  - 5:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/18273f76
```
Comment (jayvdb) [inline _reports/virresh.md]: not suitable for activity report

File: _reports/virresh.md [hunk 3]
```diff
@@ -0,0 +1,391 @@
...
+  - 6:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/9671f083
+      - details: >
+          "setup.py: Make package installable"
+  - 7:
+      - repo: coala-antlr
+      - link: https://gitlab.com/coala/bears/coala-antlr/commit/f8067834
+      - details: >
+          "Files Cleaned"
```
Comment (jayvdb) [inline _reports/virresh.md]: give this a better "details" as the commit summary is misleading

File: _reports/virresh.md [hunk 4]
```diff
@@ -0,0 +1,391 @@
...
+      - details: >
+          "report.md: Make GSoC year dynamic"
+  - 46:
+      - repo: coala-mobans
+      - link: https://gitlab.com/coala/mobans/commit/76ebb40c
+      - details: >
+          "appveyor.yml.jj2: Add template for appveyor config"
+  - 47:
+      - repo: grammars-v4
+      - link: https://github.com/antlr/grammars-v4/pull/1205
```
Comment (jayvdb) [inline _reports/virresh.md]: try to get this merged : https://github.com/antlr/grammars-v4/pull/1205#issuecomment-411577709

if it is an unmerged PR, move it to last in the list

**Comment by jayvdb** [comment]
ack https://github.com/coala/projects/pull/617/commits/03a149978d01411673e36269f64cc863d7d5c584

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**